BigBug/MainForm.cs

```diff
@@ -38,7 +38,6 @@ namespace BigBug
 {
     public partial class MainForm : Form
     {
-        Settings settings = new Settings("settings.ini"); ///< Settings wrapper to load or save settings.
```

Comment:
I believe it will be nice to turn Settings.cs into abstract and making alternative versions for windows (to keep the INI file, since it is much easier to edit) and Mono (that wraps Settings.Settings). User data file for Settings.Settings is located under appdata and less practical. https://docs.microsoft.com/en-us/dotnet/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp